frontend/src/components/APIService.js

```diff
@@ -1,71 +1,64 @@
 export default class APIService {
-  static LoginUser(body) {
+  static LoginUser({ email, password }) {
     return fetch('/api/token/login/', {
       method: 'POST',
       headers: {
         'Content-Type': 'application/json'
       },
-      body: JSON.stringify(body)
+      body: JSON.stringify({ email, password })
     }).then((resp) => resp.json());
```

Comment:
haven't noticed it previously, but you should also add in each promise an `.catch` block

e.g.:
```
}).then((resp) => resp.json())
  .catch(error => console.error(`Login user error: ${error}`)
```